README.md

```diff
@@ -16,8 +16,8 @@
         <a href="https://material.uplabs.com/posts/fancyshowcaseview">@MaterialUp Best of the Day</a>
 </p>
 
-
```

Comment:
this is ok. But other additional lines looks unnecessary.

Reply:
ooh, sorry I thought they were errors. I use markdown lint vs code extension and it highlighted them as errors

Reply:
You can check errors from codacy:
https://app.codacy.com/manual/faruktoptas/FancyShowCaseView/file/39286766552/issues/source?bid=14909056&fileBranchId=14909056

Reply:
Sure. I guess a new commit removing added lines will do?

Reply:
Yes please